ercc/README.md

```diff
@@ -14,9 +14,7 @@ committing state changes.
 
 The enclave registry is implemented as a normal chaincode and comes with a
 custom validation plugin. Before you can install ercc at a peer you have to
-build the vscc plugin by running the following.
-
-    $ make
+build the custom-vscc plugin in `$FPC_PATH/plugins/`. 
```

Comment:
you might want to explicitly link to the corresponding README given that it has special build instructions? Actually, top-level readme also might have to be correspondingly be updated?

Reply:
> BTW: i wanted also to quickly check whether ercc registration was successful. But alas, seems right now you could test with ercc you can do it only with `getAttestationReport` which requires an enclavepk as input and just copy pasting the one returned by an FPC CC query did not work :-( Would be nice to use same format or have a `list registered enclaves` call in ercc or alike. Not that this would be really on the critical path nor worth adding now with ercc anyway soon changing quite a bit ... :-)

Sounds like a good feature for the new ercc! @bvavala

Reply:
More than good...it's needed!
Short term: from chaincode id to a one-item list -- the enclave at the designated peer for a specific chaincode.
Medium term: from chaincode id to a list -- the enclaves available in the channel for that chaincode.
Long term: a list of enclaves in the channel which can load and run a chaincode.